tests/virt/node/gpu/vgpu/conftest.py

```
@@ -76,3 +89,163 @@ def hco_cr_with_node_specific_mdev_permitted_hostdevices(
+        "spec": {
+            "vgpuManager": {
+                "image": "qe-cnv-tests-ocp-nvidia-aie-vgpu-installer",
+            }
+        }
+    }
+    with ResourceEditor(patches={cluster_policy: patch_data}):
+        yield
+
+
+@pytest.fixture(scope="class")
+def update_daemon_set_to_enable_mig_vgpu(update_cluster_policy_to_enable_mig_vgpu, admin_client):
```

Ruff (0.15.10)

[warning] Unused function argument: `update_cluster_policy_to_enable_mig_vgpu`

(ARG001)